.github/workflows/openwrt-ci.yml

```
@@ -28,6 +28,7 @@ jobs:
         uses: actions/checkout@v2
         with:
           ref: master
+          fetch-depth: 1000000
```

```suggestion
          fetch-depth: 0
```

It's more reasonable to set to `0` (indicates all history).